### ISSUE TITLE
Sign text cut off problem when using TTF Fonts

### ISSUE THREAD

**Issue by gomz0717** [OpenRCT2/OpenRCT2#7153]
**OS:** [Windows 10]
**Version:** [0.1.2]
**Commit/Build:** [0e032c9]

My language setting is Korean. So, I use TTF fonts to rendering Korean.
But when I using TTF Fonts, the upper part of the SIGN text is cut off 1~3px.
At first, I tried to reduce the font size, but didn't work.
I adjusted the value of 'y_offset', but didn't work either.
I thought it was the problem because of the difference between Korean and English Fonts,
but I saw the same problem with tiny English fonts.

The below is GIFs that I tested.

**Screenshots / Video:**

![gulim11px_default](https://user-images.githubusercontent.com/36295976/36025107-39156e94-0dd5-11e8-9c63-e16d95f2911b.gif)
gulim.ttc (Korean default font) / 11px(Default size)

![gulim10px](https://user-images.githubusercontent.com/36295976/36025119-47e69c9a-0dd5-11e8-97b7-8f465ac36679.gif)
gulim.ttc (Korean default font) / 10px

![gulim9px](https://user-images.githubusercontent.com/36295976/36025121-4add649c-0dd5-11e8-808e-2570753b2d06.gif)
gulim.ttc (Korean default font) / 9px

![tinyengilshfont](https://user-images.githubusercontent.com/36295976/36025125-4d3ae584-0dd5-11e8-9272-3bee5c08eabb.gif)
04b03.ttf (Tiny English font) / 8px

**Comment by AaronVanGeffen** [comment]
The trouble is just how little space there is to work with... I ran into the same problem with Japanese. Below is a diff of the area I modified at the time, in order to make it look slightly better. How do those changes look for you?
```diff
--- a/src/openrct2/drawing/scrolling_text.c                                                                                          
+++ b/src/openrct2/drawing/scrolling_text.c                                                                                          
@@@ -1567,13 -1567,13 +1567,13 @@@ void scrolling_text_set_bitmap_for_ttf(                                                           
      const uint8 *src = surface->pixels;
  
      // Offset
--    height -= 3;
--    src += 3 * pitch;
++    height -= 2;
++    src += 2 * pitch;
      height = min(height, 8);
```

**Comment by gomz0717** [comment]
It works good!

But when I change another font or size from default (korean default font is bigger than text area, so I changed smaller font than the default), height changes. So I changed offset(ScrollingText.cpp) again and rebuild it, then works Fine with new font.

I don't know much about programming, so I don't know if it's possible. But If it's possible, I hope I can modify the font options in config file. For example, you can specify a different font each size(like OpenTTD), and can change y position of scrolling text.

(Sorry for my poor English!)

**Comment by AaronVanGeffen** [comment]
Thank you for testing. Glad to know my little hack had the intended effect. 

Yeah, this should really use the offset values passed in font descriptor instead of using a hardcoded value. I'll PR a fix soon. :)

**Comment by gomz0717** [comment]
Thank you for responding! I really appreciate it!

**Comment by AaronVanGeffen** [comment]
This should be fixed per #7154.
